exp-player/addon/components/exp-rating-form/component.js

```diff
@@ -192,8 +192,8 @@ const TEN_POINT_SCALE = [
     'number9',
     'number10'
 ];
-const SEVEN_POINT_SCALE = TEN_POINT_SCALE.slice(0, 8);
-const NINE_POINT_SCALE = TEN_POINT_SCALE.slice(0, 10);
+const SEVEN_POINT_SCALE = TEN_POINT_SCALE.slice(1, 8);
```

Comment:
These scales are different lengths before (0,8) and after (1,8). How did this work out to the same number of items??